.github/workflows/release.yml

```diff
@@ -68,20 +72,25 @@ jobs:
         name: Install protoc
         run: sudo apt update && sudo apt install -y protobuf-compiler
 
-      - name: Install Rust toolchain
+      - if: ${{ runner.os == 'Windows' }}
+        name: Install Rust toolchain
+        run: rustup toolchain install stable --profile minimal --target ${{ matrix.target }}
+
+      - if: ${{ runner.os != 'Windows' }}
+        name: Install Rust toolchain
         run: |
           curl https://sh.rustup.rs -sSf \
             | sh -s -- -y --default-toolchain stable --profile minimal --target ${{ matrix.target }}
           echo "$HOME/.cargo/bin" >> "$GITHUB_PATH"
```

Comment:
It looks like the non-Windows command downloads Rustup before invoking it, while the Windows command just invokes it. How does that work?

Reply:
The only reason we need to download rustup is because it's not available on our self-hosted runners. All the GitHub runners already have it installed